### PR TITLE
Migrate tests to GitHub Actions

### PR DESCRIPTION
**Pull request by bentranter**
Migrates the tests from Travis to GitHub Actions.

Also fixes a `gofmt` failure so that those checks would pass.

Tested at https://github.com/bentranter/doctl/pull/1.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -103,6 +105,12 @@ shellcheck:
 	@echo ""
 	@scripts/shell_check.sh
 
+.PHONY: gofmt_check
+gofmt_check:
```
Comment (andrewsomething) [inline Makefile:109]: :+1:  Thanks for this! We should probably add `go lint` at some point as well. Though the results aren't exactly pretty right now...

**Review by andrewsomething — APPROVED** [review]
:shipit:  LGTM!

**Comment by bentranter** [comment]
Travis seems stuck, so I'm gonna merge this.

File: Makefile (hunk 1 above)
Comment (bentranter) [reply Makefile:109]: Definitely, I thought about adding a ticket to address it during Hacktoberfest, but realized there were way too many warnings to ask anyone from the community to work on fixing it.

**Pull request merged**